### DISCUSSION TITLE
Support for AC Series?

### DISCUSSION THREAD

**Discussion by PeterHaban** [nathanmarlor/foxess_modbus#163] [General]
Good afternoon :)

My AC 5.0 has been delivered today and I was planning to set it up with the StealthChesnut version which now suggests this might be a better option. From what I can see this one doesn't have support for the the AC range yet, right? If you're interested in adding support for that range I'm happy to help once I get it all wired up :) What I've got is the following:
AC-5.0-E
USR W610
2x SDM230 (ESPHome + MAX13487 to monitor)
1x EC4100 Battery Master (4kWh)
2x EC4100 Battery Master (4kWh)

**Comment by canton7** [comment]
The H1/AC1 should be supported, yep

**Reply by PeterHaban** [reply]
Excellent, I'll give it a go once I have it all wired up :)
